MonoGame.Framework/Storage/StorageContainer.cs

```diff
@@ -118,9 +119,15 @@ internal StorageContainer(StorageDevice device, string name, PlayerIndex? player
 			// From the examples the root is based on MyDocuments folder
 #if WINDOWS_STOREAPP || WINDOWS_UAP
 			var saved = "";
-#elif LINUX || MONOMAC
+#elif MONOMAC
             // We already have a SaveData folder on Mac/Linux.
             var saved = StorageDevice.StorageRoot;
+#elif DESKTOPGL
+            string saved = "";
+            if(CurrentPlatform.OS == OS.Linux)
+                saved = StorageDevice.StorageRoot;
+            else

```

Comment:
Good call

-----Original Message-----
From: "KonajuGames" <notifications@github.com>
Sent: ‎13/‎05/‎2015 03:44
To: "mono/MonoGame" <MonoGame@noreply.github.com>
Cc: "Dean Ellis" <dellis1972@googlemail.com>
Subject: Re: [MonoGame] Setup storage device for DesktopGL (#3844)

In MonoGame.Framework/Storage/StorageContainer.cs:
>              // We already have a SaveData folder on Mac/Linux.
>              var saved = StorageDevice.StorageRoot;
> +#elif DESKTOPGL
> +            string saved = "";
> +            if(CurrentPlatform.OS == OS.Linux)
> +                saved = StorageDevice.StorageRoot;
> +            else
That makes sense to me.
—
Reply to this email directly or view it on GitHub.